cernopendata/templates/cernopendata_pages/visualise_events_cms.html

```diff
@@ -1,17 +1,49 @@
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/CombinedCamera.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/TrackballControls.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/Projector.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/CanvasRenderer.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/SVGRenderer.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/MTLLoader.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/OBJLoader.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/OBJExporter.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/STLExporter.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/GLTFExporter.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/jszip.min.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/DeviceOrientationControls.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/StereoEffect.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/lib/StereoCamera.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/config.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/setup.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/animate.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/files-load.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/objects-draw.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/objects-add.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/objects-config.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/geometry/dt.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/geometry/csc.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/geometry/rpc.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/controls.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/tree-view.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/display.js') }}"></script>
+<script src="{{ url_for('static', filename='node_modules/ispy-webgl/js/ispy.js') }}"></script>
```

Comment:
Since we also need to import these files in the `record_detail.html` template, perhaps we can create a couple of new templates for iSpy `ispy_js/ispy_css` and import them here and in `record_detail.html`. Also in these files, you can briefly add a comment on how to maintain/update them. What do you think?

P.S. Perhaps this splitting could be useful for the Opera visualiser so we can also document how to keep it up to date.